document/core/exec/runtime.rst

```diff
@@ -608,6 +660,66 @@ This definition allows to index active labels surrounding a :ref:`branch <syntax
    The selected label is identified through the :ref:`label index <syntax-labelidx>` :math:`l`, which corresponds to the number of surrounding |LABEL| instructions that must be hopped over -- which is exactly the count encoded in the index of a block context.
 
 
+.. index:: ! throw context, event, exception, throw address, catch block
+.. _syntax-ctxt-throw:
+
+Throw Contexts
+..............
+
+In order to specify the reduction of :ref:`try-catch blocks <syntax-try>`
+with the help of the administrative instructions |THROWADDR|, |EXNREFADDR|, and |CATCHN|,
+the following syntax of *throw contexts* is defined, as well as associated structural rules:
+
+.. math::
+   \begin{array}{llll}
+   \production{(throw contexts)} & \XT &::=&
+     \val^\ast~[\_]~\instr^\ast \\ &&|&
+     \LABEL_n\{\instr^\ast\}~\XT~\END \\ &&|&
+     \CATCHN_n\{\instr^\ast\}~\XT~\END \\ &&|&
+     \FRAME_n\{F\}~\XT~\END \\
+   \end{array}
+
+.. math::
+   \begin{array}{rcl}
+   S;~F;~\CATCHN_m~\{\instr^\ast\}~\val^m~\END &\stepto& S;~F;~\val^m \\
```

Comment:
These reduction rules seem out of place here. I think they all belong into instructions.rst, probably into an auxiliary section on exception handlers there, analogous to the ones for blocks.

Reply:
Done.